### ISSUE TITLE
use another key aside from C-h

### ISSUE THREAD

**Issue by mohkale** [justbur/emacs-which-key#228]
I'm having difficulty changing the default paging command from `C-h` to something else. I'd prefer to set it to `<C-return>` (because I'd like to keep hjkl and C-h C-j C-k C-l for my vim like bindings).

I've tried:

```elisp
(define-key which-key-mode-map (kbd "<C-return>") 'which-key-C-h-dispatch)
```

but if I enter a prefix like SPC, wait for which key to popup and then hit `C-return` I get a `SPC <C-return> is undefined`.

I tried 

```elisp
(setq which-key-use-C-h-commands nil
      which-key-paging-key "<C-return>")
```

but that also didn't work. could you clarify how we change the paging key?

**Comment by justbur** [comment]
If you wanted to use an ascii character, you could change the value of `help-char`, which might be the simplest thing to do, but this won't work for `<C-return>`. 

If you want to use `which-key-paging-key` you also need to set `which-key-paging-prefixes` as the docstring suggests. This is tedious, but it's the only alternative to using the `help-char`. The `help-char` is treated specially by emacs internally, which is why it offers an alternative to the long route of defining *all* of the prefixes you want paging to occur after.

**Comment by MoHKale** [comment]
Ok, thnx. Redefining help-char to to something else freed up `C-h`, but which key still shows `[C-h paging/help]` at the bottom. Shouldn't it update to show the current value of `help-char`?

**Comment by justbur** [comment]
Thanks. This should be fixed with 8554a9f
